### PR TITLE
Update first chain tutorial to 3e65111

### PR DESCRIPTION
**Pull request by riusricardo**
Update to Substrate 3e65111

### CODE REVIEW

File: docs/tutorials/creating-your-first-substrate-chain/build.md [hunk 1]
```diff
@@ -114,7 +114,7 @@ proof to the owner of that proof and the block number the proof was made.
 ```rust
 // This pallet's storage items.
 decl_storage! {
-    trait Store for Module<T: Trait> as PoeStorage {
+    trait Store for Module<T: Trait> as TemplateModule {
```
Comment (JoshOrndorff) [inline docs/tutorials/creating-your-first-substrate-chain/build.md]: This doesn't seem related to the DispatchResult thing. Are you making this consistent with something else?

Comment (riusricardo) [reply docs/tutorials/creating-your-first-substrate-chain/build.md]: It's not related but it's also not used. The javascript part uses it as `TemplateModule`.
